openmmml/models/anipotential.py

```diff
@@ -57,22 +57,32 @@ class ANIPotentialImpl(MLPotentialImpl):
     def __init__(self, name):
         self.name = name
 
+
     def addForces(self,
                   topology: openmm.app.Topology,
                   system: openmm.System,
                   atoms: Optional[Iterable[int]],
                   forceGroup: int,
                   filename: str = 'animodel.pt',
+                  implementation : str = 'nnpops',
+                  overwrite_periodicity : Optional[bool]=False,
                   **args):
+        """
+        overwrite_periodicity : whether to overwrite the periodicity given by `topology` or `system` to its opposite; this is primarily for 
+            testing purposes
+        """
         # Create the TorchANI model.
 
         import torchani
         import torch
         import openmmtorch
+
+        # `nnpops` throws error if `periodic_table_index`=False if one passes `species` as `species_to_tensor` from `element`
+        _kwarg_dict = {'periodic_table_index': True} #if implementation == 'nnpops' else {}
```

Comment:
Was that code at the end meant to be commented out?  If so, we should probably remove it to avoid confusion.

Reply:
yeah this should be removed.